html/elements/a.json

```
@@ -351,6 +351,54 @@
             }
           }
         },
+        "implicit_noopener": {
+          "__compat": {
+            "description": "Setting <code>target=\"_blank\"</code> also provides implicit <code>rel=\"noopener\"</code> behavior",
```

Thank you for this Chris. The data is good. I had a thought that the description could be a bit punchier, like this:

```suggestion
            "description": "<code>target=\"_blank\"</code> implies <code>rel=\"noopener\"</code> behavior",
```

Less wordy still would be this…

```suggestion
            "description": "<code>target=\"_blank\"</code> implies <code>rel=\"noopener\"</code>",
```

…but I'm not sure it's right: `someElement.getAttribute("rel") === "noopener"` but there's no actual `rel="noopener"` attribute. I'm not sure if that's a distinction without a difference. I'd be interested in your take.